hms-app/src/components/card-details/IdeaDetails.tsx

```diff
@@ -132,6 +132,212 @@ export default function IdeaDetails(props: IProps) {
     </div>
   ))
 
+
+  // Idea Card Functions
+  const ideaName = () => {
```

Comment:
maybe ideaHeader would be a better name as it contains the owner pictue, name and idea title